__tests__/e2e/package.json

```diff
@@ -5,7 +5,12 @@
   "license": "MIT",
   "scripts": {
     "setup": "bash scripts/setup.sh",
-    "test": "npm i && npm run setup &&  bash scripts/test.sh"
+    "test": "npm i && npm run setup &&  bash scripts/test.sh",
+    "setup_integration": "bash scripts/setup_integration.sh",
```

Comment:
the ideal way in NPM scripts is to use colons `setup:integration`, `test:integration` etc.